src/evaluations/report_generator.py

```diff
@@ -155,9 +155,9 @@ def _format_num_estimable_sets_df(df):
       df['num_estimable_sets_cell'] = (
           df[analyzer.NUM_ESTIMABLE_SETS].astype('str')
           + '<br>relative_error: mean='
-          + df[simulator.RELATIVE_ERROR + '_mean'].round(4).astype('str')
+          + df[simulator.RELATIVE_ERROR_BASENAME + '1_mean'].round(4).astype('str')
```

Comment:
I know you didn't write this code, but I do wonder if the precision of the rounding should be a flag or at least a constant defined at top?

Reply:
Even though every CS 101 class teaches that one should never have constants in the code as above, this is one case where I think the utility of defining a manifest constant is debatable.  Nonetheless, I have made the change you requested.